etc/apache2/vhosts.d/openqa-common.inc

```diff
@@ -49,11 +49,21 @@ ProxyPass /stylesheets !
 ProxyPass /assets !
 ProxyPass /error !
 
+# ensure websocket connections are handled as such by the reverse proxy while
```

Comment:
I assume the comment relates to `RewriteEngine On`? Would it hurt to call this command twice? Usually I prefer to enable/include stuff where I need it rather than relying that this is done by something else anyways.

Reply:
Yep, it's about the engine on thingie, but it's more a comment than an actual need to change anything